### ISSUE TITLE
Add functionality to create client applications and contracts

### ISSUE THREAD

**Issue by zekucukkose** [apiman/apiman-cli#15]
The CLI currently does not provide a command to create client applications or contracts between client applications and apis.

It would be a useful feature if these things could be created using the declarative mode (similar to how plugins etc can be configured from a yaml or json file).

**Comment by outofcoffee** [comment]
Agreed, @zekucukkose this would be a good feature addition. Would you be in a position to submit a PR for this? Happy to discuss approach/architecture.

**Comment by jquantin** [comment]
have a look here please : https://github.com/jquantin/apiman-cli/commit/7f0a1818c88063f0312be921b0ef35285b9737cd
